auth-sample/build.gradle

```diff
@@ -53,7 +53,7 @@ android {
 }
 
 dependencies {
-    implementation 'com.spotify.android:auth:1.1.0'
+    implementation 'com.spotify.android:auth:1.1.1'
```

Comment:
I'd almost say this should be 1.2.0, since it contains breaking changes. What do you think?

Reply:
yes, definitely.